modules/network-observability-quickfilter.adoc

```diff
@@ -7,6 +7,13 @@
 = Filtering the network traffic
 By default, the Network Traffic page displays the traffic flow data in the cluster based on the default filters configured in the `FlowCollector` instance. You can use the filter options to observe the required data by changing the preset filter.
 
+Query Options::
+You can use *Query Options* to optimize the search results, as listed below:
+
+** *Log Type*:  A conversation is an aggregation of flows between the same peers. The available options *Conversation* and *Flows* provide the ability to query flows by events, such as new connection, completed connection, periodic record with updates for long connections and flow logs. 
```

Comment:
Think `end connection` sounds better than completed, but I will leave it to @jpinsonneau for the final say

Reply:
I agree that `end connection` sounds/reads better.